config/routes.rb

```diff
@@ -7,7 +7,7 @@
   # /merchants/:merchant_id/dashboard
   resources :merchants, only: [] do
     resources :dashboard, module: "merchant", only: [:index]
-    resources :items, module: "merchant", only: [:index, :show, :edit, :update]
+    resources :items, module: "merchant"
```

Comment:
Technically, we have everything but a destroy now
`resources :items, module: "merchant", except [:destroy]`